docs/managed-victoriametrics/setup-notifications.md

```diff
@@ -0,0 +1,104 @@
+---
+sort: 6
+weight: 6
+title: Notifications in Managed VictoriaMetrics
+menu:
+  docs:
+    parent: "managed"
+    weight: 6
+---
+# Notifications in Managed VictoriaMetrics
+
+This section add ability to setup channels for sending alerting notifications.
```

Comment:
The guide covers how to enable email and Slack notifications

Reply:
updated